app/src/main/res/values/styles.xml

```diff
@@ -341,17 +341,16 @@
 
     <style name="SimpleDraweeViewPlaceholder">
         <item name="actualImageScaleType">centerCrop</item>
-        <item name="placeholderImage">?attr/thumbnail_image_placeholder</item>
         <item name="placeholderImageScaleType">fitCenter</item>
-        <item name="failureImage">?attr/thumbnail_image_placeholder</item>
+        <item name="failureImage">?android:colorBackground</item>
         <item name="failureImageScaleType">fitCenter</item>
```

Comment:
Since the `failureImage` is only a general color, so set up `failureImageScaleType` is unnecessary.